execute/executor.go

```diff
@@ -94,21 +93,20 @@ func (e *executor) createExecutionState(ctx context.Context, p *plan.PlanSpec, a
 		deps:      e.deps,
 		alloc:     a,
 		resources: p.Resources,
-		results:   make(map[string]flux.Result, len(p.Results)),
+		results:   make(map[string]flux.Result, len(p.Roots())),
```

Comment:
I think I like the name `Results` better as its unambiguous... it could be possible someone interprets `Roots` as the sources.